client-src/elements/chromedash-header.ts

```diff
@@ -150,6 +150,11 @@ export class ChromedashHeader extends LitElement {
   @state()
   loading = false;
 
+  private _redirectToCurrentPage(): void {
```

Comment:
Question, is void ok or never preferable? 

https://www.typescriptlang.org/docs/handbook/2/functions.html#never